build-tools/gradle/gradlew

```diff
@@ -65,6 +65,9 @@ case "`uname`" in
 esac
 
 CLASSPATH=$APP_HOME/gradle/wrapper/gradle-wrapper.jar
+if [ -z ${JAVA_HOME+x} ] && [ -d "~/Library/Android/jdk" ]; then
+  JAVA_HOME="$HOME/Library/Android/jdk"
+fi
```

Comment:
Let's try to remove this diff entirely, I think it should be fixed by https://github.com/xamarin/xamarin-android/commit/9fd888d02c1743035f6dc39da7c96a507259cd6b

Reply:
Resolved